### PR TITLE
Ansible lazy

### PR DESCRIPTION
**Pull request by dmendes-daitan**
Just adding some checks that gave me errors :)

### CODE REVIEW

File: dev-setup/tasklist/host-after.yml [hunk 1]
```diff
@@ -1,7 +1,13 @@
 ########
 # Dev Env setup script for Developer machines
 ########
+
+- name: "Check if Android Studio is installed"
```
Comment (smarkovik) [inline dev-setup/tasklist/host-after.yml]: Is this needed ? homebrew will just skip it... 
I don't mind, just curious...

btw liking the fact you're ramping up your ansible skills :)

File: dev-setup/tasklist/host-after.yml [hunk 2]
```diff
@@ -40,12 +46,19 @@
             untracked = cyan
 
 
+- name: "Check if a bash directory exists"
```
Comment (smarkovik) [inline dev-setup/tasklist/host-after.yml]: what's the `.bash` folder?

File: dev-setup/tasklist/host-after.yml (hunk 1 above)
Comment (dmendes-daitan) [reply dev-setup/tasklist/host-after.yml]: I already had and the cask returned a error instead of just passing by. That's why I've added a check

Comment (dmendes-daitan) [reply dev-setup/tasklist/host-after.yml]: It's the folder used for the "https://github.com/SymphonyOSF/mobile-dev-setup/blob/f87120a38927f4f4fdb503850e8f66c255710819/dev-setup/tasklist/host-after.yml#L46"

Comment (smarkovik) [reply dev-setup/tasklist/host-after.yml]: hmm.. I know why the error was thrown, it was installed manually. 
Makes sense ... thanks

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# IntelliJ files
+.idea
```
[the remark this answers is not in the document]
Comment (dmendes-daitan) [reply .gitignore]: Going back and forth on that one, maybe one day I'll say goodbye to IJ ! :)

**Pull request merged**